jenkins/test/testsuite/Jenkinsfile

```diff
@@ -34,6 +34,11 @@ pipeline {
                                             name: 'test_run_id',
```

Comment:
AFAIR, test_run_id is the jenkins `build_number`. Should that be set as default value for `test_run_id` param? That is not something a user will or should set

Reply:
Coming from the code: https://github.com/opensearch-project/opensearch-build/blob/main/src/run_integ_test.py#L33. `test_run_id` is passed like this. It's safe to remove it from the param.

Reply:
Pushed the change.

Reply:
It is being retrieved from TestArgs() https://github.com/opensearch-project/opensearch-build/blob/9bc0a41493d99b2c03f5193ca5d0437b6609907f/src/test_workflow/test_args.py#L34

Reply:
I dont think it should be removed. How will we know which test results are from which workflow run. Or is that information not relevant?

@peternied @dblock thoughts? Also don't know why we added `uuid.uuid4().hex` for test_run_id.

Reply:
Are you suggesting a path like `bundlebuild/${opensearch_version}/${build_id}/${platform}/${architecture}/test_run_id/test-results/test_type/component`?

Reply:
Nope that is take care of in test_recorder. Ask is just set the `$BUILD_NUMBER` as default value of `test_run_id` in jenkins parameters.

Reply:
I have removed `test_run_id` from Jenkinsfile as test_run_id is already being set in the code mentioned above.

Reply:
As discussed offline with @gaiksaya. I'll change the path to `bundlebuild/${opensearch_version}/${build_id}/${platform}/${architecture}/test_run_id/test-results/test_type/component` to record BUILD_NUM as the test_run_id